test/ShapeCrawler.Tests.Unit/ShapeTests.cs

```diff
@@ -582,4 +582,96 @@ public void Text_Setter_updates_shape_text()
         // Assert
         shape.Text.Should().Be("Test");
     }
+
+    [Test]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 1 Round 0.25", "0.20834")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 2 Round 0.25", "0.20834")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 3 Round 0.25", "0.20834")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 1 Round 0", "0.0")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 1 Round X", "0.35")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 1 Round 1", "1.0")]
+    [SlideShape("057_corner-radius.pptx", 1, "Size 1 Round 0.75", "0.61112")]
+    public void CornerRadius_getter_returns_values(IShape shape, string expectedRoundednessStr)
+    {
+        // Arrange
+        decimal? expectedRoundedness = string.IsNullOrEmpty(expectedRoundednessStr) ? null : decimal.Parse(expectedRoundednessStr);
```

Comment:
`expectedRoundednessStr` is always non-`null`. Can be just `decimal? expectedRoundedness = decimal.Parse(expectedRoundednessStr);`?

Reply:
Right, this is a hold-over from when I thought getter would return null in the case where no roundedness was specified.

89e8854d5cdc08c6327552b87711d973edae2df9